Makefile

```diff
@@ -13,12 +13,20 @@ PLATFORMS ?= linux_amd64 linux_arm64
 -include build/makelib/common.mk
 
 # ====================================================================================
-# Setup Images
+# Setup Output
 
-# even though this repo doesn't build images (note the no-op img.build target below),
-# some of the init is needed for the cross build container, e.g. setting BUILD_REGISTRY
--include build/makelib/image.mk
-img.build:
+# S3_BUCKET ?= crossplane.releases
+-include build/makelib/output.mk
+
+# ====================================================================================
+# Setup Helm
+
+HELM_BASE_URL = https://charts.crossplane.io
```

Comment:
I wonder who can push helm charts there?

Reply:
This is handled by Jenkins, which has the credentials to push the chart. You can see more here https://crossplane.io/docs/v0.12/release/release-process